panel/src/components/Dialogs/SearchDialog.vue

```diff
@@ -182,52 +65,8 @@ export default {
 .k-search-dialog {
 	--dialog-padding: 0;
 	--dialog-rounded: var(--rounded);
-	overflow: visible;
```

Comment:
`Question:` Anyone recall what this was for? With it the border-radius has no effect. Removing it, I din't spot any problems.

Reply:
I think it was about the outline for the input/dropdown button.